tools/release-scripts/test.js

```diff
@@ -16,10 +9,10 @@ function lint() {
 }
 
 function testAndLint() {
-  return Promise.all([
-    test(),
-    lint()
-  ]);
+  console.log('Running: '.cyan + 'eslint and tests'.green);
+
+  return exec('npm run test')
+    .then(() => console.log('Completed: '.cyan + 'eslint and tests'.green));
```

Comment:
Unfortunately in the release process we need to do the version bump before building so the new version number get's built into the docs.